tests/lora/test_lora_layers_peft.py

```diff
@@ -1092,6 +1093,68 @@ def test_get_list_adapters(self):
                 {"unet": ["adapter-1", "adapter-2", "adapter-3"], "text_encoder": ["adapter-1", "adapter-2"]},
             )
 
+    @require_peft_version(peft_version="0.6.2")
+    def test_simple_inference_with_text_lora_unet_fused_multi(self):
+        """
+        Tests a simple inference with lora attached into text encoder + fuses the lora weights into base model
+        and makes sure it works as expected - with unet and multi-adapter case
+        """
+        for scheduler_cls in [DDIMScheduler, LCMScheduler]:
```

Comment:
Can we put the schedulers into a `parameterized.expand` instead of using a `for` loop?

Reply:
Hmm I personally find parameterzied.expand hard to debug and deal with, also note that for loop is present on all other tests, so maybe that could be potentially done in a follow up PR to enhance the tests but I personally prefer to keep the tests simple and avoid using parameterized when it is possible to do so

Reply:
I see, I hadn't checked the other tests above, so now I agree it's better like this for consistency. Elsewhere in diffusers, `parameterized.expand` is being used, so I think the argument of it being harder to debug doesn't quite apply. In fact, I think it's easier in the long term because, when using a loop and we get a test failure with the first scheduler, we don't know if the second scheduler would pass. So we don't know if the error is scheduler-specific or not. With `parameterized`, both schedulers would be tested, even if the first fails, so we can debug easier.

But as you said, let's leave it like this for now and core maintainers can decide if they want to rely more on `parameterized` in the future or not.

Reply:
`parameterized.expand ` is totally fine here. IMO it does help a bit to isolate a failure for a specific scheduler.

Reply:
Actually. Is there a specific reason to loop over the schedulers? I think it's sufficient to run these tests with just DDIM.

Reply:
To test LCM LoRA I think. @patrickvonplaten could you confirm?